sw/host/rom_ext_image_tools/Cargo.toml

```diff
@@ -20,3 +20,7 @@ debug = true
 
 [dependencies]
 ring = "0.16.16"
+serde = "0.8"
```

Comment:
~~Shouldn't you be using `serde = "1.0"`? (Same for `serde_derive`).~~

Uh, apparently `serde-hjson` uses a version of serde that's waaay out of date... that's kind of really bad because it will cause some unpleasant problems when libraries that use `1.0` versions get pulled in (due to differences in traits). I think we should leave this for now but I think we should come up with a long term solution, which I think means either:
- Sending them patches to fix this.
- Forking `serde-hjson`.